.github/pull_request_template.md

```diff
@@ -19,7 +20,7 @@
 - [ ] I have commented hard-to-understand code
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] All tests pass locally with my changes 
-      (Check [README.md #Contributing](https://github.com/norlab-ulaval/libpointmatcher/tree/develop#contributing)
+      (Check [contributing_instructions.md](https://github.com/norlab-ulaval/libnabo/tree/develop/doc/contributing/contributing_instructions.md)
```

Comment:
The link is pointing to libnabo, shouldn't it be libpointmatcher?

Reply:
@boxanm This is outdated. It was fixed in commit [6de0d75](https://github.com/norlab-ulaval/libpointmatcher/pull/550/commits/6de0d7585c6aa5e6283ed53de7482f1790281d52)